cmd/policy.go

```diff
@@ -47,6 +47,10 @@ var policysListCmd = &cobra.Command{
 			}
 		}
 
+		if socketID != "" {
+			policiesPath += fmt.Sprintf("&socket_id=%s", socketID)
```

Comment:
is it the first argument?  if so,  should be the `?` signal instead of `&`

Reply:
No, it will always be the last one

Reply:
cool